content/blog/scheduling-in-react/index.md

```diff
@@ -210,34 +210,34 @@ function SearchBox(props) {
 }
 ```
 
-If we take another look at the Performance tab with this change and zoom towards the end, we will see that our analytics are now sent after all rendering work has completed and so the total work in our app is perfectly scheduled:
+If we take another look at the Performance tab with this change and zoom toward the end, we’ll see that our analytics are now sent after all rendering work has completed, and so the total work in our app is perfectly scheduled:
 
-![Screenshot of Chrome DevTools that show that that React breaks down the rendering work into small chunks. Analytics are sent at the end after all rendering work has completed.](devtools-normal-and-low.png)
+![Screenshot of Chrome DevTools that show that React breaks the rendering work down into small chunks. Analytics are sent at the end after all rendering work has completed.](devtools-normal-and-low.png)
 
 Try it out:
 
 <iframe src="https://codesandbox.io/embed/v0lxm0xlzl?fontsize=14&hidenavigation=1&view=preview" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe>
 
 ## Limitations of the Scheduler
 
-With the Scheduler, it is possible to control when certain callbacks can be executed. It is built deep into the latest React implementation and works out of the box with Concurrent mode.
+With the Scheduler, it’s possible to control when certain callbacks can be executed. It is built deep into the latest React implementation and works out of the box with Concurrent mode.
 
-There are two limitations of the Scheduler though:
+That said, tere are two limitations of the Scheduler:
 
 1. **Resource Fighting.** The Scheduler tries to use all of the resources available. This causes issues if multiple instances of a scheduler run on the same thread and compete for resources. We need to ensure that all parts of our application will use the same instance.
 2. **Interleaving tasks with browser work.** Since the Scheduler runs in the browser, it can only do what the browser allows us to. Document lifecycles like rendering or garbage collection can interfere with the work in an uncontrollable way.
 
-To remove those limitations, the Google Chrome team is working together with React, Polymer, Ember, Google Maps, and the Web Standards Community to create a [Scheduling API in the browser](https://github.com/spanicker/main-thread-scheduling). What an exciting time!
+To remove these limitations, the Google Chrome team is working together with React, Polymer, Ember, Google Maps, and the Web Standards Community to create a [Scheduling API in the browser](https://github.com/spanicker/main-thread-scheduling). What an exciting time!
```

Comment:
May I ask why you changed those to these here? Is the "proximity" in this case achieved by the limitations being written the line before? 

[Here](https://en.oxforddictionaries.com/usage/these-or-those) it says that "that" should be used for things that the speaker previously mentioned so I'm confused. 

I'm pretty sure I mess up this/that all the time.

Reply:
Think about if you're talking to me about the limitations and pretend they're concrete things. You have them in your hand and you're showing them to me. And you say "these limitations (here in my hand XYZ." Versus if the limitations were across the street in the park and you were pointing far away saying "those limitations over there are XYZ." In the same way, the conversation is about the limitations that are currently close by and currently being talked about, so the "proximity" you mention is that they're immediate, right there. So you would use "these" because you're referring to something immediate/in the vicinity, so to speak. Sorry if that's mega confusing.

We could also say "To remove the above limitations..."

Reply:
Makes sense. 👍